src/lib/deploy.ts

```diff
@@ -387,6 +389,23 @@ export default class Deploy {
     if (this.logs) countdown.stop();
     await this.cache.save(colors);
 
+    // save manifest.arkb to user dir
+    if (!isFile) {
+      const dir = this.txs[0].filePath || this.txs[1].filePath;
+      const {
+        tx: { id },
+      } = this.txs.find((i) => i.type === 'application/x.arweave-manifest+json');
+      // non-necessary: but add check incase of funny/unexpected behavior
+      if (id) {
+        // find manifest json from temp dir
+        const mPath = path.resolve(getTempDir(), `${id}.manifest.json`);
+        try {
+          fs.copyFileSync(mPath, path.join(path.dirname(dir), 'manifest.arkb'));
```

Comment:
I used the `/tmp/.arkb` folder like a caching system kind of stuff.. since we're only saving the manifest after the deploy is done. and the only place I can save the manifest file is in the `buildManifest` method. so I just temporarily stored it in the temp folder and copy / rename it to the upload directory.